doc/source/examples/analysis/alignment_and_rms/rmsd.ipynb

```diff
@@ -447,7 +451,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can also calculate the weighted RMSD of a trajectory using a custom array of weights. (Note: up until version 0.21.0, you cannot calculate the weighted RMSD of ``groupselections``.)"
+    "You can also calculate the weighted RMSD of a trajectory using a custom array of weights. The `weights` keyword only applies to the group in `select`, and does not apply to the `groupselections`; these are unweighted."
```

Comment:
That's a little bit confusing, I guess you introduce `weights_groupselections` further down, do you think it's worth bringing up here?